packages/manager/modules/bm-server-components/src/general-information/installation/ovh/server-installation-ovh.controller.js

```diff
@@ -496,9 +496,9 @@ export default class ServerInstallationOvhCtrl {
       this.$stateParams.productId,
     ).then((spec) => {
       this.$scope.informations.diskGroups =
-        spec.diskGroups?.sort((a, b) =>
-          a.description.localeCompare(b.description),
-        ) || [];
+        spec.diskGroups
+          ?.sort((a, b) => a.description.localeCompare(b.description))
+          .filter((diskGroup) => diskGroup.raidController !== 'cache') || [];
```

Comment:
Maybe you should filter before sorting so you sort only the filtered array and not the entire one